docs/DeveloperGuide.adoc

```diff
@@ -358,6 +358,16 @@ image::UndoRedoActivityDiagram.png[width="650"]
 ** Cons: Requires dealing with commands that have already been undone: We must remember to skip these commands. Violates Single Responsibility Principle and Separation of Concerns as `HistoryManager` now needs to do two * different things.
 // end::undoredo[]
 
+===== Aspect: Support for multiple themes
+* **Alternative 1**: Do not support multiple themes
+** Pros: Consistent user experience with a single theme.
+** Cons: No ability to customize the look and feel of the application.
+* **Alternative 2**: Support multiple themes.
+** Pros: Users get a certain extent of customizability in the application.
+** Cons: More development effort needed to support multiple themes and ensure that they can be loaded.
```

Comment:
This sounds like simply considering implementing the feature or not. Instead, it should be a consideration faced during implementation. For example, customizable vs preset, or fully themeable vs just changing colours.